presto-main/src/main/java/com/facebook/presto/connector/ConnectorManager.java

```diff
@@ -196,7 +196,7 @@ private synchronized void addCatalogConnector(String catalogName, ConnectorId co
 
         MaterializedConnector informationSchemaConnector = new MaterializedConnector(
                 createInformationSchemaConnectorId(connectorId),
-                new InformationSchemaConnector(catalogName, nodeManager, metadataManager, accessControlManager));
+                new InformationSchemaConnector(catalogName, nodeManager, metadataManager, accessControlManager, transactionManager));
```

Comment:
This change and `InformationSchemaConnector.java` doesn't appear in https://github.com/prestosql/presto/pull/90/commits/7baabd1f658a6eccfeec61101f61cbaf3c972d13 :). Maybe again get dropped on PrestoSQL side.

Reply:
Yeah, good catch